eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -1,5 +1,12 @@
 <UsageData>
   <IgnorePatterns>
-    <UsagePattern IdentityGlob="*/*" />
+    <UsagePattern IdentityGlob="Microsoft.SourceBuild.Intermediate.*/*" />
+
+    <!-- Ignore System.Text.JSon 7.0.1 and dependencies until we can add
+         7.0 targeting pack and these packages to SBRP.
+         https://github.com/dotnet/source-build/issues/3330 -->
+    <UsagePattern IdentityGlob="System.Text.Json/7.0.1" />
+    <UsagePattern IdentityGlob="Microsoft.Bcl.AsyncInterfaces/7.0.0" />
+    <UsagePattern IdentityGlob="System.Text.Encodings.Web/7.0.0" />
```

Comment:
Curious, why can't those be live references from runtime instead?

Reply:
Because we should not introduce more places where we need to worry about coherency :)

Reply:
I thought that live dependencies are always preferred over SBRP (for obvious reasons). deployment-tools is part of the VMR. Is another node repo node in the graph that bad?

Reply:
It's bad in context of the current build. That means every time we have a servicing patch, preview, etc. We have yet another parallel path to flow runtime through before we can release the product. We only really need to depend on live dependencies where the consuming repo repackages meaningful info about the dependency. For instance, the SDK will repackage the actual binaries, so it needs to depend on the latest. Consider roslyn, it targets net6 in it's build, but inserts into the SDK. It rolls forward to run on the SDK runtime.